### ISSUE TITLE
Longitudes are not constrained to -180..180

### ISSUE THREAD

**Issue by stevage** [umap-project/umap#395]
I ran into this problem while exporting data.  The GeoJSON file contained longitudes like -215, instead of +145. Now I see that you can actually create distinct points at different multiples of 360 degrees:

![longwrap](https://cloud.githubusercontent.com/assets/678344/19374886/fbe175ea-921a-11e6-9887-ac5839156546.png)

At the very least, the longitudes should be normalised to -180..+180 when exporting GeoJSON.

**Comment by yohanboniface** [comment]
There are edge cases, see https://github.com/Leaflet/Leaflet/pull/1293 :/

> At the very least, the longitudes should be normalised to -180..+180 when exporting GeoJSON.

Yes, let's do that anyway.

**Comment by stevage** [comment]
Yikes! What a gnarly problem. You could solve all the easy cases though,
everything except a line or polygon that straddles the date line.

**Comment by davidbgk** [comment]
Somehow, I think it has been solved by https://github.com/umap-project/umap/pull/1168 @yohanboniface ?

**Comment by yohanboniface** [comment]
Nope, that commit only deals with coordinates set manually, because of Leaflet/Leaflet#1293 I'm not sure we can't totally block longitudes > 180 (say one want to put a circumnavigation by boat for example…). :(
